project/Settings.scala

```diff
@@ -32,7 +32,13 @@ object Settings {
     val scalaXml            = "1.2.0"
     val catsTime            = "0.4.0"
 
-    val http4sVersion  = "1.0.0-M27"
+    val http4sVersion                  = "0.23.14-27-fd0137d-SNAPSHOT"
```

Comment:
We needed the snapshot for an unreleased feature, we can hold this PR until the next release.

Reply:
Because Observe is a work in progress, a long way away for being in production, I say it is acceptable to have it depend on a snapshot temporarily.